helm/Chart.yaml

```diff
@@ -22,3 +22,4 @@ annotations:
     - Allow wildcard in ingress host
     - Remove smtp default example values
     - Add unknownExpireAfter in management-api configuration
```

Comment:
@sebdevaux for the next 3.20.x, we should remove these 3 lines and keep only:
`Allow users to define extra manifests`
WDYT?

Reply:
Agree